crates/ibc-core/ics24-host/src/utils.rs

```diff
@@ -0,0 +1,45 @@
+use core::time::Duration;
+
+pub fn calculate_block_delay(
+    delay_period_time: &Duration,
+    max_expected_time_per_block: &Duration,
+) -> u64 {
+    let delay_period_time = delay_period_time.as_secs();
+    let max_expected_time_per_block = max_expected_time_per_block.as_secs();
+    if max_expected_time_per_block == 0 {
+        return 0;
+    }
+    if delay_period_time % max_expected_time_per_block == 0 {
+        return delay_period_time / max_expected_time_per_block;
+    }
+    (delay_period_time / max_expected_time_per_block) + 1
```

Comment:
we can use [`u64::div_ceil`](https://doc.rust-lang.org/stable/std/primitive.u64.html#method.div_ceil)

Reply:
Let's address this as an optimization PR later.